pkg/hcl2nix/export.go

```diff
@@ -22,6 +22,10 @@ type OCIArtifact struct {
 	ImportConfigs []string `hcl:"importConfigs,optional"`
 	// DevDeps defines if development dependencies should be present in the image. By default, it is false.
 	DevDeps bool `hcl:"devDeps,optional"`
+	// Base defines if the OCI artifact that needs to be built is for a Base image. By default, it is false.
+	Base bool `hcl:"base,optional"`
```

Comment:
I think we can avoid having these params. 

We can assume that if OCI block is `pkgs` , then it is a base image. 
We can also assume that if they don't pass `DevDeps` as true(from CLI flag), we'll only add runtime packages.